google/cloud/security/common/gcp_api/compute.py

```diff
@@ -47,23 +47,20 @@ def __init__(self, credentials=None, version=None):
     # TODO: Migrate helper functions from gce_firewall_enforcer.py
     # ComputeFirewallAPI class.
 
-    def get_backend_services(self, project_id):
-        """Get the backend services for a project.
+    def _get_paged_list(self, resource_name, list_request, list_next_request):
```

Comment:
This can be done later, but could you put this in the base client class and call it something like

_build_paged_aggregated_list() or something (there's a method in that class that does this for list_next, but doesn't handle aggregatedList_next).